Source/DataSources/StaticGroundPolylinePerMaterialBatch.js

```diff
@@ -59,6 +61,8 @@ define([
         this.subscriptions = new AssociativeArray();
         this.showsUpdated = new AssociativeArray();
         this.zIndex = zIndex;
+
+        this._asynchronous = defaultValue(asynchronous, true);
```

Comment:
Small thing, but it doesn't look like we would expect this input to ever be undefined. Should be safe to just set it directly.